packages/client/components/AnalyticsPage.tsx

```diff
@@ -93,7 +94,7 @@ const AnalyticsPage = () => {
     if (gaMeasurementId) {
       ReactGA.initialize(gaMeasurementId, {
         gtagOptions: {
-          send_page_view: true
```

Comment:
By removing this line, we fall back to the [default setting](https://github.com/PriceRunner/react-ga4/blob/f31ed502adec6dc9dd68e3b59d39c1857880f433/src/ga4.js#L190), which is `send_page_view = false`. This is because we are going to manually send the `page_view` event on Line 196.

Reply:
-1 I'm not sure if it's this part, but I get each page view event twice, only once with the `content_group` set
https://www.loom.com/share/62de4f1c77b448d28c3fcf956598e34d